### ISSUE TITLE
Bug in Relationship filtering v0.3.22

### ISSUE THREAD

**Issue by w01fgang** [keystonejs/keystone#3892]
Filtering doesn't work as expected. Some items can't be found.
Each time I type my text the query remains the same
`/keystone/api/autos/autocomplete?context=relationship&q=&list=offers&field=cars`

The problem is in `buildOptionQuery` method
```
buildOptionQuery: function buildOptionQuery(input) {
		var value = input && input[0] && input[0].value || '';
		var filters = this.buildFilters();
		return 'context=relationship&q=' + value + '&list=' + Keystone.list.path + '&field=' + this.props.path + (filters ? '&' + filters : '');
	},
```
result `value` is empty string

### Steps to reproduce the behavior
Typing text in input field of relationship
### Expected behavior
Shows items list filtered by query from all items from DB.
### Actual behavior
Shows items list filtered by query but for the first 50 items from DB

**Comment by w01fgang** [comment]
It seems to be fixed in repo.
When you plan to make next release?

**Comment by thejameskyle** [comment]
If it looks fixed in the repo, it's probably because it's fixed in 4.0.

We're not planning on back-porting changes in v4 to v3, so I'm going to close this
